plugins/main/public/components/common/data-grid/data-grid-service.ts

```diff
@@ -198,6 +198,7 @@ export const onFilterCellActions = (
   setFilters: (filters: Filter[]) => void,
 ) => {
   return (field: string, operation: FILTER_OPERATOR, value?: any) => {
+    if (!value) return;
```

Comment:
:nitpick**: this could ignore falsy values. Check how this is managed in OpenSearch Dashboards. We could be ignoring some use cases.